wiki/Tournaments/COT/2022/en.md

```diff
@@ -0,0 +1,88 @@
+| Wiki editor | ![][flag_RU] [Nelly](https://osu.ppy.sh/users/4741164) |
+
+## Links
+
+- **[Main spreadsheet](https://docs.google.com/spreadsheets/d/1iie_zXj1GbS_Sx4MyWdGOYD9raofZ46uomMbVW6wm48/edit?usp=sharing)**
+- [Discussion thread](https://osu.ppy.sh/community/forums/topics/1493650)
+- [Discord server](https://discord.gg/ZqCdWBKGrM)
+- [Challonge brackets](https://challonge.com/ru/cot22)
+- [Livestream channel](https://www.twitch.tv/playosuru/)
+
+## Ruleset
+
+### General rules
+
+1. Map scoring is based on **[ScoreV2](/wiki/Score#scorev2).**
+2. The match lobby will be created by the referee, and the players will be invited.
+3. Players have a strict 10-minute grace period available to play the match. If 10 minutes pass and a player does not show up the match, the other player will receive a win by default.
+4. Warm ups aren't allowed.
+5. During the lobby it is not allowed to show any form of harassment during livestreaming. By organizers after breaking the rule the player may be disqualified.
+6. At the start of the match, both players roll. The roll winner decides to ban or pick first.
+7. The referee is not allowed to play in the match, which means the referee should not be in the lobby during the match process.
+8. Each player is allowed 1 or 2 bans per match. Banned maps are removed from the mappool for the duration of the match and not allowed to be picked. A player is not allowed to ban 2 maps from the same mod pool.
+9. Players have 2 minutes to select a map; if they fail to choose in the allotted time, the map choice will be random.
+10. Each player may use a tactical timeout with the 2 minute length which can be used once during the tournament. The tactical timeout is optional and it is not required to use it.
+11. After choosing a map players have a 2 minute preparation. A referee will start the match when both players are ready. After that, when the timer ends, referee starts the match.
+12. NF is forced on every map.
+13. In the case of a disconnection: if the disconnection happened in the first 30 seconds during the match, it will be replayed. If the disconnection happened after 30 seconds, then the score won't count.
+
+### Qualifiers
+
+1. All the rules found above will be in effect during the qualifier lobbies. Qualifier lobbies will be Head to Head and there are no bans or warmups.
+2. In the lobby, the players will play through the entire mappool in the order given on the mappool sheet. This means you will play through the NM maps, then HD, then HR, and then DT.
+3. You will play through the pool once.
+4. In the event of any disconnects, the player who disconnects will be allowed to replay the map after the run by themselves.
+5. Qualifier scores will be hidden until all players have played, in order to prevent seed manipulation. Which means that the player placing 32th place will play against the player placing 1st place etc.
```

Comment:
```suggestion
2. In the lobby, the players will play through the entire mappool once, in the order given on the mappool sheet. This means you will play through the NM maps, then HD, then HR, and then DT.
3. In the event of any disconnects, the player who disconnects will be allowed to replay the map after the run by themselves.
4. Qualifier scores will be hidden until all players have played, in order to prevent seed manipulation, which means that the player placing 32nd will play against the player placing 1st, etc.
```